tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -428,10 +486,18 @@ describe(__filename, () => {
 
     sinon.assert.calledWith(
       fakeRouter.push,
-      `/${lang}/${clientApp}/collections/${username}/${slug}/`
+      `/${newLang}/${clientApp}/collections/${username}/${slug}/`
     );
   });
 
+  it('calls router.goBack() on cancel when creating', () => {
```

Comment:
Not sure if it's worth it to test that we *don't* call `goBack()` when not creating 🤷‍♂️

Reply:
Well , we do have a test above for `redirects to the detail view on cancel when editing` which checks that `rotuer.push()` is called when not creating, which is kind of the same thing. I suppose it doesn't prove that both `router.goBack()` and `router.push()` are not called, but that doesn't seem like something we'd normally test against.